scripts/templates/results.html

```diff
@@ -27,7 +27,17 @@
             <div class="fl w-80 pa2">
                 <span lang="gez">{{ result.incipit_txt_gez|safe }}</span><br/>
                 <!-- source mss -->
-                <span class="id fl purple avenir">{{ result.source_s }}</span>
+                <span class="id fl purple avenir">{{ result.source_s }}</span><br>
```

Comment:
Do we need the `<br>` ? Not a huge objection, just would prefer to avoid if we can.

Reply:
I tried a few ways around this, but couldn't find anything straightforward